packages/flutter/lib/src/material/chip.dart

```diff
@@ -1944,9 +1944,10 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
       ),
     );
     return Semantics(
+      button: widget.tapEnabled,
```

Comment:
This seems like the right property to for this:
```
  /// For example, the [Chip] class sets this to false because it can't be
  /// disabled, even if no callbacks are set on it, since it is used for
  /// displaying information only.
```

Looking good =)